include/odp/api/spec/traffic_mngr.h

```diff
@@ -556,19 +586,20 @@ void odp_tm_egress_init(odp_tm_egress_t *egress);
 
 /** Query All TM Capabilities
  *
- * The odp_tm_capabilities() function can be used to obtain the complete set of
- * TM limits supported by this implementation.  The reason that this returns
- * a SET of capabilities and not just one, is because it is expected that
- * many HW based implementations may have one set of limits for the HW and
- * also support a SW TM implementation with a (presumably larger) different
- * set of limits.  There are also cases where there could be more than
- * SW implementation (one supporting say tens of thousands of tm_queues and
- * a variant supporting tens of millions of tm_queues).
+ * This function returns the complete set of TM capabilities for all egresses.
```

Comment:
Now that I read this I think the wording I suggested may give the wrong impression that the array parameter is there so that the function can return a capa struct for every egress. So I would like to propose another rewording for the opening sentence: "This function returns the set of TM capabilities that are common for all egresses."